test/watchCases/plugins/automatic-prefetch-plugin/0/index.js

```diff
@@ -0,0 +1,8 @@
+it("should watch for changes", function() {
+	require("./foo/" + WATCH_STEP).should.be.eql('This is only a test.' + WATCH_STEP);
+    if(WATCH_STEP > 0){
```

Comment:
space between ) and {

convert WATCH_STEP active to an number. It's a string. Currently this relies on automatic conversion from string to number... uncool.

``` js
if(+WATCH_STEP > 0) {
```